concepts/contexts.py

```diff
@@ -711,6 +711,25 @@ def shape(self) -> _common.Shape:
             Shape(objects=6, properties=10)
         """
         return _common.Shape._from_pair(self.objects, self.properties)
+    
+    @tools.lazyproperty
+    def density(self) -> float:
+        """The density of the context.
+
+        Returns:
+            Context density.
+        
+        Example:
+            >>> import concepts
+            >>> context = concepts.Context(['cat', 'dog', 'bird'],
```

Comment:
Any concern about usig `concepts.EXAMPLE` (cf. the other examples)?

Reply:
I have tried `concepts.EXAMPLE`, problem is that `fill_ratio` for `concepts.EXAMPLE` is `0.5`. I would prefer some example which won't be exactly `0.5` `fill_ratio` to demonstrate the feature better.

I will check other examples in your docstrings.

Reply:
I see, we can load two differen contexts in the doctest example in that case (I think its nice for the reader to give all the outputs for the same example context here).

Reply:
I would welcome more example contexts like `concepts.EXAMPLE` in general. If am I understand right, the `concepts.EXAMPLE` is from linguistic domain which is for me personally hard to understand. I can prepare some small example context from for example animal domain (maybe based on https://github.com/mikulatomas/zoo-typicality/tree/pre-publish).